pyccl/emulators/emu_base.py

```diff
@@ -0,0 +1,60 @@
+__all__ = ('Emulator', )
+
+from abc import abstractmethod
+
+class Emulator(object):
```

Comment:
Since we may have emulators of other quantities in the future, could we call this `EmulatorPk` or something like that?

Reply:
Agree, changed it